README.md

```diff
@@ -1,15 +1,27 @@
-# WinUI 3 Gallery 
+![WinUI Gallery hero image](README_Images/WinUIGalleryLanding.PNG)
 
+<h1 align="center">
+    WinUI 3 Gallery
+</h1>
+
+<p align="center">
 [![Build Status](https://dev.azure.com/ms/microsoft-ui-xaml/_apis/build/status%2Fmicrosoft.WinUI-Gallery?branchName=main)](https://dev.azure.com/ms/microsoft-ui-xaml/_build/latest?definitionId=612&branchName=main) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/your/your-project/blob/master/LICENSE)

```

Comment:
I think adding this p-tag broke the links in our readme.

Reply:
Most likely. Mixing HTML and MD can do that.

Reply:
Opening that URL shows a (error) message:

> {"count":1,"value":{"Message":"Build pipeline microsoft.WinUI-Gallery was not found."}}